docs/examples/testing_pillar/crud/event_generation.py

```diff
@@ -0,0 +1,39 @@
+import json
+from typing import Any, Dict, Optional
+
+
+def generate_api_gw_event(
+    body: Optional[Dict[str, Any]] = None,
+    path_params: Optional[Dict[str, Any]] = None,
+) -> Dict[str, Any]:
+    return {
```

Comment:
after the deck is ready, we could use `hypothesis` (property based testing) for this part and have an awesome example